app/routes/rental_routes.py

```diff
@@ -0,0 +1,111 @@
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from flask import Blueprint, jsonify, request, abort
+from datetime import datetime, timedelta
+from app.routes.video_routes import check_for_valid_input
+
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+def validate_input(request_body):
```

Comment:
yaaay helper methods. Like before, maybe turning this into a boolean return would work best